packages/api-plugin-discounts/package.json

```diff
@@ -34,5 +34,10 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "scripts": {
+    "test": "jest --passWithNoTests",
```

Comment:
We should remove this `passWithNoTests` because it should not pass with no tests